File: volume1/chapter6/lambda_palindrome.js
exports.handler = (event, context, callback) => {
	const input = event.word.replace(/ /g, '').toLowerCase();
	const reverse = input.split('').reverse().join('');
	const is_palindrome = input == reverse;
const response = is_palindrome ? `'${event.word}' is a palindrome!` : 
`'${event.word}' is not a palindrome.`;
	callback(null, response);
}

File: volume1/chapter6/lambda_palindrome.py
import json

def lambda_handler(event, context):
    input = event["word"]
    data = input.lower().replace(' ', '')
    reverse = data[::-1]
    is_palindrome = data == reverse
    response = "'{0}' is a palindrome!".format(input) if is_palindrome else "'{0}' is not a palindrome".format(input)
    return response

